eyeshade/controllers/accounts.js

```diff
@@ -77,6 +77,54 @@ ORDER BY created_at
   }
 }
 
+/*
+   GET /v1/accounts/balances/top
```

Comment:
#486 is mostly concerned about channel balances, not owner or uphold.

How about making this `GET /v1/accounts/balances/{type}/top` so that publishers will end up calling `GET /v1/accounts/balances/channel/top`